core/ui/PageTemplate/MessageHandler/tm-modal.tid

```diff
@@ -0,0 +1,10 @@
+title: $:/yaisog/ui/PageTemplate/MessageHandler/tm-modal
```

Comment:
:) ... IMO needs to be `/core/ui`

Reply:
This is temporary, since I'd assume the $messagecatcher will not be permanently used to redirect tm-modal calls. I didn't want to imply that this should be added to the core. The title has no meaning for functionality.